rest/management-api-v2/PostType.curl

```diff
@@ -33,7 +33,12 @@ curl --request POST \
      "type": "rich_text",
      "content_group": {
        "external_id": "article-copy"
-     }
+     },
```

Comment:
Looks like you added the default value for a rich text element. But default values aren't yet supported for rich text elements.

Reply:
yey, stupid mistake, sorry :)